charts/prometheus-pingmesh-exporter/values.yaml

```diff
@@ -0,0 +1,396 @@
+    port: http
+
+nodeSelector: {}
+tolerations: 
+  - operator: Exists
+    effect: NoSchedule
+  - key: CriticalAddonsOnly
+    operator: Exists
+  - operator: Exists
+    effect: NoExecute
+  - key: node.kubernetes.io/not-ready
+    operator: Exists
+    effect: NoExecute
+  - key: node.kubernetes.io/unreachable
+    operator: Exists
+    effect: NoExecute
+  - key: node.kubernetes.io/disk-pressure
+    operator: Exists
+    effect: NoSchedule
+  - key: node.kubernetes.io/memory-pressure
+    operator: Exists
+    effect: NoSchedule
+  - key: node.kubernetes.io/pid-pressure
+    operator: Exists
+    effect: NoSchedule
+  - key: node.kubernetes.io/unschedulable
+    operator: Exists
+    effect: NoSchedule
+    
+affinity: {}
+
+# if the configuration is managed as secret outside the chart, using SealedSecret for example,
+# provide the name of the secret here. If secretConfig is set to true, configExistingSecretName will be ignored
+# in favor of the config value.
+configExistingSecretName: ""
+# Store the configuration as a `Secret` instead of a `ConfigMap`, useful in case it contains sensitive data
+secretConfig: false
+config:
+  modules:
+    http_2xx:
+      prober: http
+      timeout: 10s
+      http:
+        valid_http_versions: ["HTTP/1.1", "HTTP/2"]
+        valid_status_codes: [200,201,204,304,312]
+        method: GET
+        preferred_ip_protocol: "ip4"
+    http_2xx_ipv6:
+      prober: http
+      timeout: 10s
+      http:
+        valid_http_versions: ["HTTP/1.1", "HTTP/2"]
+        valid_status_codes: [200,201,204,304,312]
+        method: GET
+        preferred_ip_protocol: "ip6"
+    http_post_2xx:
+      prober: http
+      timeout: 10s
+      http:
+        valid_http_versions: ["HTTP/1.1", "HTTP/2"]
+        valid_status_codes: [200,201,204,304,312]
+        method: POST
+        preferred_ip_protocol: "ip4"
+    http_post_2xx_ipv6:
+      prober: http
+      timeout: 10s
+      http:
+        valid_http_versions: ["HTTP/1.1", "HTTP/2"]
+        valid_status_codes: [200,201,204,304,312]
+        method: POST
+        preferred_ip_protocol: "ip6"
+    tcp_connect:
+       prober: tcp
+       timeout: 2s
+    ping:
+      prober: icmp
+      timeout: 5s
+      icmp:
+        preferred_ip_protocol: "ip4"
+    ping_ipv6:
+      prober: icmp
+      timeout: 5s
+      icmp:
+        preferred_ip_protocol: "ip6"
+    dns:
+       prober: dns
+       dns:
+         transport_protocol: "tcp"
+         preferred_ip_protocol: "ip4"
+         query_name: "kubernetes.defalut.svc.cluster.local"
+    dns_ipv6:
+       prober: dns
+       dns:
+         transport_protocol: "tcp"
+         preferred_ip_protocol: "ip6"
+         query_name: "kubernetes.defalut.svc.cluster.local"
+
+pinglistConfig:
+  setting:
+    concurrent_limit: 20
+    interval: 60.0
+    delay: 200
+    timeout: 2.0
+    source_ip_addr: 0.0.0.0
+    ip_protocol: ip6
+  mesh:
+    add-ping-public: 
+      name: ping-public-demo
+      type: OtherIP
+      ips :
```

Comment:
can we please remove the chinese urls and ips here?

Reply:
Done. Remove the chinese urls and ip